specification/dartLangSpec.tex

```diff
@@ -5837,8 +5839,13 @@ \section{Extensions}
 \item $D$ declares a type parameter named \code{E}.
 \item $D$ declares a member whose basename is the name of a type parameter
   of $D$.
-\item $D$ declares a member (static or not) with the same basename as a
-  member declared by the built-in class \code{Object}.
+\item $D$ declares an instance member or a static member whose basename is
+  \code{hashCode}, \code{noSuchMethod}, \code{runtimeType}, or \code{toString}.
+  Or $D$ declares an operator named \lit{==}.
```

Comment:
(Is this necessary, or could we just say "member whose basename is ... \lit{==}", because that is the name of the member introduced by the `operator==` declaration?)

Reply:
It is true, we do define the basename of operators, and they are instance members. I just changed the text from a form that relied on this to a form where the operator is mentioned separately, because I thought it would be easier to read. However, taking another look, `\lit{==}` works fine, too. Done.